.github/workflows/cli-test-npm.yml

```diff
@@ -0,0 +1,38 @@
+name: CLI Tests on npm installations
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to use for running test cases'
+        required: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x, 14.x, 10.x]
+    steps:
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+         node-version: ${{ matrix.node-version }}
+      -  name: Global - Testing installation of Twilio CLI (1)
+         run: |
+          npm install -g twilio-cli@${{ github.event.inputs.version }}
+          node src/asserts/validate_tests.js
+      - name: Global - Testing upgrade Twilio CLI (2)
+        run: |
+         npm install -g twilio-cli@${{ github.event.inputs.version }}
+            npm install -g twilio-cli
+            node src/asserts/validate_tests.js
+      - name: Local - Testing installation of Twilio CLI (1)
+        run: |
+         npm install twilio-cli
+         node src/asserts/validate_tests.js
+      - name: Local - Testing upgrade Twilio CLI (2)
+        run: |
+         npm install twilio-cli@${{ github.event.inputs.version }}
+          npm install twilio-cli
```

Comment:
Similar to the comments above

Reply:
To test different scenarios I've updated `make install` in the above test case and kept this one same.